app/Livewire/Project/Database/Health.php

```
@@ -0,0 +1,81 @@
+        try {
+            $this->authorize('update', $this->database);
+            $this->syncData(true);
+            $this->dispatch('success', 'Health check updated. Restart the database to apply the changes.');
+        } catch (\Throwable $e) {
+            return handleError($e, $this);
+        } finally {
+            if (is_null($this->database->config_hash)) {
+                $this->database->isConfigurationChanged(true);
+            } else {
+                $this->dispatch('configurationChanged');
+            }
```

PHPMD (2.15.0)

[error] The method submit uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)